packages/sui-js/README.md

```diff
@@ -5,6 +5,22 @@
 $ npm install @s-ui/js --save
 ```
 
+## pipe
+Consist of a chain of processing functions, where the output of each element is the input of next
+
+```js
+import pipe from @s-ui/js/lib/pipe
+
+execute = async search => {
+  const pageName = await this.pipe(
+    this._fromQueryStringToQueryParamsPageMapper.map,
+    this._fromQueryParamsToTargetPageValueObjectPageMapper.map
+  )(search)
+
+  return pageName
+}
```

Comment:
not sure if this is a good example as only those familiar with our domain stuff will know what that mapper functions should look like and what search is. It would be better if you could add an example that works by copy and paste it

Reply:
It's true, I change it! Thank you!